dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/thread/Stopper.java

```diff
@@ -14,26 +14,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package org.apache.dolphinscheduler.common.thread;
 
 import java.util.concurrent.atomic.AtomicBoolean;
 
 /**
- *  if the process closes, a signal is placed as true, and all threads get this flag to stop working
+ * if the process closes, a signal is placed as true, and all threads get this flag to stop working
  */
 public class Stopper {
 
-	private static AtomicBoolean signal = new AtomicBoolean(false);
-	
-	public static final boolean isStopped(){
-		return signal.get();
-	}
-	
-	public static final boolean isRunning(){
-		return !signal.get();
-	}
-	
-	public static final void stop(){
-		signal.set(true);
-	}
+    private static AtomicBoolean signal = new AtomicBoolean(false);
+
+    public static final boolean isStopped() {
+        return signal.get();
+    }
+
+    public static final boolean isRunning() {
+        return !signal.get();
+    }
+
+    public static final void stop() {
+        signal.set(true);
+    }
+
+    /**
+     * should only be used for test
+     */
+    public static final void reset() {
```

Comment:
```suggestion
    public static final void reset() {
```

Suggest do not to write this method just for test, and it also will cause the system error if we use this method in the project.

Reply:
reverted.